guides/source/security.md

```diff
@@ -1024,6 +1024,24 @@ Environmental Security
 
 It is beyond the scope of this guide to inform you on how to secure your application code and environments. However, please secure your database configuration, e.g. `config/database.yml`, and your server-side secret, e.g. stored in `config/secrets.yml`. You may want to further restrict access, using environment-specific versions of these files and any others that may contain sensitive information.
 
+Search Engines Indexing
+-----------------------
+
+Sometimes you don't want that some pages of your application be visible on search sites like Google,
```

Comment:
How about?

> To prevent pages of your application from being visible on search site like Google,

Otherwise:

> Sometimes, you don't want some pages of your...